streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorStateManager.java

```diff
@@ -224,9 +224,6 @@ public void close(Map<TopicPartition, Long> ackedOffsets) throws IOException {
             checkpoint.write(checkpointOffsets);
         }
 
-        // close the restore consumer
-        restoreConsumer.close();
-
```

Comment:
Closing the consumer used for restoring state was moved to [here](https://github.com/confluentinc/kafka/pull/57/files#diff-045aeaddb4232a85a8560186b4901e69R207), which is in the same class that now creates the consumer.